entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# if lnd enabled, attempt to connect
+if [[ -z "${LND_RPC_AUTHORITY}" ]]
+then ./bin/agora --directory files --http-port $AGORA_PORT --lnd-rpc-authority $LND_RPC_AUTHORITY --lnd-rpc-cert-path .lnd/tls.cert --lnd-rpc-macaroon-path .lnd/invoices.macaroon
```

Comment:
Since podman and docker can be used to map host ports to container ports, can agora used a fixed port inside the container?

Ideally port 80, since that's the standard web service port. If that requires configuration, because it's a privileged port, than port 8080 is good to.

If that works, then `AGORA_PORT` doesn't need to be configurable.

Reply:
This makes sense